src/transaction.rs

```diff
@@ -513,16 +431,17 @@ impl Transaction {
     /// Returns all existing keys within the specified range, including soft-deleted
     /// and thus hidden by tombstones.
     /// The returned keys are not added to the read set and will not cause read-write conflicts.
-    pub fn keys_with_tombstones<'b, R>(&'b self, range: R) -> Result<Vec<Vec<u8>>>
+    pub fn keys_with_tombstones<'b, R>(&'b self, range: R) -> impl Iterator<Item = &'b [u8]> + 'b
```

Comment:
I don't think ' + 'b' is needed here; same goes for another three instances in this file.

Reply:
Thanks for pointing it out